src/Apps/Artwork/Components/ArtworkPageBanner.tsx

```diff
@@ -49,7 +49,10 @@ export const ArtworkPageBanner: FC<ArtworkPageBannerProps> = props => {
       return <ArtworkUnavailableBanner />
     }
 
-    if (!partnerOffer || partnerOffer.internalID !== expectedPartnerOfferID) {
+    if (
+      me &&
```

Comment:
That is fine I think but I was under impression that we want to ask collectors to log in in case they are trying to view artwork with offer link and happen to be not logged in. This will just skip rendering the banner but will not render request to log in, right?

But I guess if the offer is really expired it might be annoying to log in just to see the expired banner  :(

Reply:
It will render the log in, the logic is in a different file :)
<img width="1740" alt="Screenshot 2024-06-24 at 14 09 14" src="https://github.com/artsy/force/assets/7518671/3879eb99-d968-4c8c-ae3c-4cef40107290">

Reply:
LGTM then! Looks good to me :)

Reply:
Ah, I was confused about this. Good catch and easy fix.